src/index.js

```diff
@@ -424,7 +424,7 @@ function createClass(obj) {
 		cl.defaultProps = obj.defaultProps;
 	}
 	if (obj.getDefaultProps) {
-		cl.defaultProps = obj.getDefaultProps();
+		cl.defaultProps = obj.getDefaultProps.bind(cl)();
```

Comment:
Would be much cheaper to use `.call()` here:

```js
cl.defaultProps = obj.getDefaultProps.call(cl);
```

Reply:
Good call ;) I've changed it to `.call`.